### PR TITLE
add docstring examples to Encoder repetition penalty logits processor

### PR DESCRIPTION
**Pull request by rajveer43**
# What does this PR do?

Related issue: #24783 

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Comment by gante** [comment]
Hi @rajveer43 👋 

This logits processor has to be applied to an encoder-decoder model, like T5. I'd like to ask for a very short example: a 20 line example would be perfect :)

**Comment by gante** [comment]
Hi @rajveer43 👋 

As per [this comment](https://github.com/huggingface/transformers/issues/24783#issuecomment-1693225365), I will no longer accept this PR. Thank you for participating, and my apologies for the inconvenience :)
